This patch was original added in aports commit
439b449b68ebf3462a3f659804c41195e2d8d8ae. It
seems to be needed since this sanity checks
fails spuriously when cros compiling fakeroot.

diff -upr fakeroot-1.27.orig/configure.ac fakeroot-1.27/configure.ac
--- fakeroot-1.27.orig/configure.ac	2022-01-30 10:43:52.992260063 +0100
+++ fakeroot-1.27/configure.ac	2022-01-30 10:46:22.989526929 +0100
@@ -25,50 +25,6 @@ AC_CACHE_CHECK([which IPC method to use]
 	       [ac_cv_use_ipc],
 	       [ac_cv_use_ipc=sysv])
 
-if test $ac_cv_use_ipc = "sysv"; then
-  AC_MSG_CHECKING([whether SysV IPC message queues are actually working on the host])
-
-  AC_LANG_PUSH(C)
-  AC_RUN_IFELSE([AC_LANG_SOURCE([[
-#include <stdlib.h>
-#include <sys/types.h>
-#include <sys/ipc.h>
-#include <sys/msg.h>
-#include <time.h>
-#include <unistd.h>
-
-int main() {
-
-  srandom(time(NULL)+getpid()*33151);
-  key_t msg_key = random();
-  int msg_get = msgget(msg_key, IPC_CREAT|0600);
-
-  if (msg_get==-1) {
-    return 1;
-  } else {
-    msgctl(msg_get, IPC_RMID, NULL);
-    return 0;
-  }
-
-}]])],[ac_cv_use_ipc=sysv],[ac_cv_use_ipc=tcp],[ac_cv_use_ipc=cross])
-
-  if test $ac_cv_use_ipc = cross; then
-    if test "$host_os" = linux-gnu; then
-      ac_cv_use_ipc=sysv
-      AC_MSG_RESULT([cross, guessing yes])
-    else
-      (set -o posix; set)
-      AC_MSG_ERROR([cross compiling, unknown result for $host_os])
-    fi
-  elif test $ac_cv_use_ipc = "tcp"; then
-    AC_MSG_RESULT([No, using TCP])
-  else
-    AC_MSG_RESULT([Yes])
-  fi
-
-  AC_LANG_POP(C)
-fi
-
 AC_ARG_WITH([dbformat],
 	    AS_HELP_STRING([--with-dbformat@<:@=DBFORMAT@:>@],
 			   [database format to use: either inode (default) or path]),
